<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSetWiget</class>
 <widget class="QWidget" name="SystemSetWiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTsCalibrate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>触屏校准</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>795</width>
     <height>758</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>795</width>
      <height>758</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnVersions">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>版本信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLicenseInfo">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>认证</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNetWork">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>网络</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdmin">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpgrade">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>相机升级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTaskFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>260</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>任务文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScriptFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>300</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>脚本文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToolFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>添加工具</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
